### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by csukuangfj**
Fix typos in the comments and documentation.

### CODE REVIEW

**Review by gramalingam — APPROVED** [review]

**Review by thiagocrepaldi — APPROVED** [review]

**Comment by gramalingam** [comment]
@csukuangfj you need to signoff the DCO before the PR can be merged. Please see https://github.com/onnx/onnx/pull/5944/checks?check_run_id=21698077110

**Comment by csukuangfj** [comment]
> @csukuangfj you need to signoff the DCO before the PR can be merged. Please see https://github.com/onnx/onnx/pull/5944/checks?check_run_id=21698077110

Done.

**Pull request merged**
